Rakefile

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require 'html-proofer'
+require 'jekyll'
+require 'dotenv/load'
+
+JEKYLL_BASENAME = ENV['JEKYLL_BASENAME'] || 'docs'
+
+task :set_untranslated do
+  puts 'Shim untranslated Japanese pages'
+  sh './scripts/shim-translation.sh jekyll/_cci2 jekyll/_cci2_ja'
+end
+
+task :build do
+  puts 'build Jekyll'
+  # Create jekyll config file to override things
+  sh "echo 'baseurl: \"\/#{JEKYLL_BASENAME}\"' > jekyll/_config_override.yml"
+  # Build the Jekyll site
+  config = Jekyll.configuration({ 
+    'source' => './jekyll', 
+    'destination' => "./jekyll/_site/#{JEKYLL_BASENAME}",
+    'config' => ['./jekyll/_config.yml','./jekyll/_config_production.yml','./jekyll/_config_override.yml']
+  })
+  site = Jekyll::Site.new(config)
+  Jekyll::Commands::Build.build site, config
+end
+
+task :make_mock_api do
+  puts 'Make mock API docs'
+  sh "echo \"<html lang=''><head><link href='/docs/assets/img/icons/favicon.png' rel='icon' type='image/png' /></head></html>\" > ./jekyll/_site/index.html"
+  sh "mkdir -p ./jekyll/_site/docs/api/2.0/"
+  sh "mkdir -p ./jekyll/_site/docs/api/v2/"
+  sh "cp ./jekyll/_site/index.html ./jekyll/_site/docs/api/2.0/index.html"
+  sh "cp ./jekyll/_site/index.html ./jekyll/_site/docs/api/v2/index.html"
+  sh "cp ./jekyll/_site/index.html ./jekyll/_site/docs/api/index.html"
+  sh "rm ./jekyll/_site/index.html"
+end
```

Comment:
What is :make_mock_api used for / where is it being implemented in the `config.yml`?

Reply:
To successfully run the `html-proofer` test locally, we need to build API as well, but I want to skip that step.
So, this is a workaround to skip building API docs. 

I created another task `build_local`, this is the command for preparing `html-proofer` test.
https://github.com/circleci/circleci-docs/pull/5141/files#diff-ee98e028c59b193d58fde56ab4daf54d43c486ae674e63d50ddf300b07943e0fR38